packages/react-mui-store-file-viewer/src/components/CreateNewShareButton/CreateSharesButton/ShareAuthenticatorField.tsx

```diff
@@ -29,266 +31,243 @@ interface FieldProps {
 }
 
 interface AuthenticatorValue {
+  shareType: string;
   authenticators: ObjectStore.ShareAuthenticator[];
   authenticatorType: string | undefined;
 }
 
 const UnControlledShareAuthenticatorField: FC<
   FieldComponentProps<FieldProps>
 > = ({ onValueChange, ...props }) => {
-  const { configurations } = useStoreFileViewerContext();
-
   const value: AuthenticatorValue = props.value;
   const theme = useTheme();
+  const { contentType, password, loading } = useCreateShareContext();
   const [authenticatorType, setAuthenticatorType] =
     useState<ObjectStore.ShareAuthenticatorType>(
       ObjectStore.ShareAuthenticatorType.Sms
     );
+  const minHeight =
+    value.shareType === "link" && contentType === "file"
```

Comment:
What is a shareType of link? password?

Reply:
Ok, I see it is for switching between the "email" view and the "get link" view

Reply:
yep

Reply:
updated the changelog